### PR TITLE
Grass turfs and a few bug fixes

### PR DESCRIPTION
**Pull request by SamHPurp**
Adds Shazbot's grass sprites, and makes them randomly select at runtime.

Adds spawn points for Birdtalons new job roles.

Enables wall smoothing for the wooden walls.

image

### CODE REVIEW

**Comment by SamHPurp** [comment]
@shazbot194 would be good to get some dirt path tiles with your grass on the edge, so that they tile together well.

File: code/game/turfs/simulated/planet.dm [hunk 1]
```diff
@@ -4,7 +4,12 @@
  planetary_atmos = TRUE
 
 /turf/open/floor/planet/grass
-	icon_state = "grass"
+  icon = 'icons/turf/floors/planet.dmi'
+  icon_state = "grass-1"
+
+/turf/open/floor/planet/grass/Initialize()
+  . = ..()
+  icon_state = "grass-[rand(1, 4)]"
```
Comment (FlattestGuitar) [inline code/game/turfs/simulated/planet.dm]: Why are there just two spaces in all of these and why does it compile?

Comment (SamHPurp) [inline code/game/turfs/simulated/planet.dm]: My atom is being funky? It wasn't 2 spaces in mine!

**Comment by FlattestGuitar** [comment]
👌

**Pull request merged**